recipes/go/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "go" %}
+{% set version = "1.9.1" %}
+{% set p = 'm2-' if win else '' %}
+{% set n = 'm2w64-' if win else '' %}
+{% set sha256 = "a84afc9dc7d64fe0fa84d4d735e2ece23831a22117b50dafc75c1484f1cb550e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://storage.googleapis.com/golang/go{{ version }}.src.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - 0001-Skip-unshare-and-capset-tests-on-Linux.patch  # [linux]
+    - 0001-Fix-for-TestRespectSetgidDir.patch  # [osx]
+    - 0001-Skip-flaky-tests-on-Windows.patch  # [win]
```

Comment:
Might want to raise issues upstream if there aren't already issues. Wouldn't hurt to link relevant issues here to make it easier for you to keep track of. :)